exp/query/config.go

```diff
@@ -26,8 +26,12 @@ type Config struct {
 
 	VectorizerAddr string `long:"vectorize-addr" description:"vectorizer address to be used to vectorize near-text query" default:"0.0.0.0:9999"`
 
+	// MetadataGRPCAddress is the host which will be used to connect to the metadata node's gRPC server.
+	// Note that this should be replaced later to avoid having a single metadata node as a single point of failure.
+	// If MetadataGRPCAddress is empty, the querier will connect to localhost.
+	MetadataGRPCAddress string `long:"metadata.grpc.address" description:"metadata grpc address" default:":9050"`
 	// AlwaysFetchObjectStore flag ignore what version does local querier has and fetch from
 	// object store (source of truth) all the time.
 	// NOTE: Enabling this (without any intermediate cache) can introduce more latency. Can be used to during performance testing, debugging, correctness check, etc.
-	AlwaysFetchObjectStore bool `long:"always-fetch-objectstore" description:"always fetch from object storage during query, skip local querier state." default:"false"`
+	AlwaysFetchObjectStore bool `long:"always-fetch-objectstore" description:"always fetch from object storage during query, skip local querier state."`
```

Comment:
when trying to run this before, i got the error:

```sh
./weaviate --target=querier
FATA[0000] failed to parse command line args             app=weaviate err="boolean flag `always-fetch-objectstore' may not have default values, they always default to `false' and can only be turned on"
```

removing the default fixed it

Reply:
thanks Nate.